var/spack/repos/builtin/packages/glproto/package.py

```diff
@@ -18,5 +18,7 @@ class Glproto(AutotoolsPackage, XorgPackage):
 
     version("1.4.17", sha256="9d8130fec2b98bd032db7730fa092dd9dec39f3de34f4bb03ceb43b9903dbc96")
 
+    depends_on("c", type="build")  # generated
```

Comment:
@DarylGrunau I assume this, and following directives, were not `# generated`:
```suggestion
    depends_on("c", type="build")
```
? :slightly_smiling_face:

Reply:
Yep, I'll fix them all and push again